### PR TITLE
Added construct workflow documentation

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: Amit Kumar Das <amit@chaosnative.com>

**What this PR does / why we need it**:
This PR adds:
- Construct workflow docs

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [ ] Signed the commit for DCO to be passed
-   [ ] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

File: website/docs/user-guides/construct-workflow.md [hunk 1]
```diff
@@ -4,4 +4,253 @@ title: Construct Workflow YAML without ChaosCenter
...
+        command:
+          - sh
+          - -c
+        args:
+          - "kubectl delete chaosengine -l 'instance_id in
+            (86a4f130-d99b-4e91-b34b-8f9eee22cb63, )' -n
+            {{workflow.parameters.adminModeNamespace}} "
+```
+
+The structure of a chaos workflow is similar to that of a Kubernetes Object. It consists of the mandatory fields like `apiVersion`, `kind`, `metadata`, `spec`.
```
Comment (oumkale) [inline website/docs/user-guides/construct-workflow.md]: Can we please explain its components and after that workflow example?

File: website/docs/user-guides/construct-workflow.md [hunk 2]
```diff
@@ -4,4 +4,253 @@ title: Construct Workflow YAML without ChaosCenter
...
+
+## Resources
+- The ChaosExperiment CR and ChaosEngine CR of different experiments are available at **[ChaosHub](http://hub.litmuschaos.io/)**.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/dan0AXO_soY?start=310" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/xLjTx8lqTuQ?start=1163" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Learn More
```
Comment (oumkale) [inline website/docs/user-guides/construct-workflow.md:257]: Add experiment and results also

File: website/docs/user-guides/construct-workflow.md [hunk 3]
```diff
@@ -4,4 +4,253 @@ title: Construct Workflow YAML without ChaosCenter
...
+- [What is a Chaos Workflow](../concepts/chaos-workflow)
+- [What is a Chaos Workflow](../concepts/chaos-engine)
```
Comment (oumkale) [inline website/docs/user-guides/construct-workflow.md]:
```suggestion
- [What is a Chaos Engine](../concepts/chaos-engine)
```

File: website/docs/user-guides/construct-workflow.md [hunk 4]
```diff
@@ -4,4 +4,253 @@ title: Construct Workflow YAML without ChaosCenter
...
+A basic chaos workflow consists of these steps: 
+1. Install ChaosExperiment
+2. Install ChaosEngine
+3. Revert Chaos
+
+## Before we begin
+To construct a Chaos Workflow without ChaosCenter, make sure you are aware of [Chaos Workflow](../concepts/chaos-workflow) and the different steps present in it.  
+
+## Steps to Construct a Chaos Workflow
+LitmusChaos leverages the popular workflow and GitOps tool **Argo** to achieve this goal. Argo enables the creation of different chaos scenarios together in from of workflows which are extremly simple and efficient to use.<br/>
```
Comment (oumkale) [inline website/docs/user-guides/construct-workflow.md]:
```suggestion
LitmusChaos leverages the popular workflow and GitOps tool **Argo** to achieve this goal. Argo enables the orchestrating of different chaos scenarios together in form of workflow which is extremely simple and efficient to use.<br/> 
```

File: website/docs/user-guides/construct-workflow.md [hunk 5]
```diff
@@ -4,4 +4,253 @@ title: Construct Workflow YAML without ChaosCenter
...
+```yaml
+      templates:
+    - name: custom-chaos
+      steps:
+        - - name: install-chaos-experiments
+            template: install-chaos-experiments
+        - - name: pod-delete
+            template: pod-delete
+        - - name: revert-chaos
+            template: revert-chaos
```
Comment (oumkale) [inline website/docs/user-guides/construct-workflow.md]: Will you please add one more example for parallel steps?

File: website/docs/user-guides/construct-workflow.md (hunk 4 above)
Comment (S-ayanide) [reply website/docs/user-guides/construct-workflow.md]: *orchestration

Comment (S-ayanide) [reply website/docs/user-guides/construct-workflow.md]: *in the form of

File: website/docs/user-guides/construct-workflow.md [hunk 6]
```diff
@@ -4,4 +4,253 @@ title: Construct Workflow YAML without ChaosCenter
...
 
-## Coming soon
+**Chaos Workflow** is a set of different operations coupled together to achieve desired chaos imapact on a Kubernetes Cluster. <br/>
+A basic chaos workflow consists of these steps: 
+1. Install ChaosExperiment
+2. Install ChaosEngine
+3. Revert Chaos
+
+## Before we begin
+To construct a Chaos Workflow without ChaosCenter, make sure you are aware of [Chaos Workflow](../concepts/chaos-workflow) and the different steps present in it.
```
Comment (S-ayanide) [inline website/docs/user-guides/construct-workflow.md]: We should add Chaos Engine too as a Concept they should know before beginning

File: website/docs/user-guides/construct-workflow.md [hunk 7]
```diff
@@ -4,4 +4,253 @@ title: Construct Workflow YAML without ChaosCenter
...
+3. Revert Chaos
+
+## Before we begin
+To construct a Chaos Workflow without ChaosCenter, make sure you are aware of [Chaos Workflow](../concepts/chaos-workflow) and the different steps present in it.  
+
+## Steps to Construct a Chaos Workflow
+LitmusChaos leverages the popular workflow and GitOps tool **Argo** to achieve this goal. Argo enables the creation of different chaos scenarios together in from of workflows which are extremly simple and efficient to use.<br/>
+
+Here is a sample pod-delete chaos workflow.
+```yaml
```
Comment (S-ayanide) [inline website/docs/user-guides/construct-workflow.md:71]: can we please add embedmd to always stay in sync with the updated yaml!

File: website/docs/user-guides/construct-workflow.md [hunk 8]
```diff
@@ -4,4 +4,253 @@ title: Construct Workflow YAML without ChaosCenter
...
+
+## Resources
+- The ChaosExperiment CR and ChaosEngine CR of different experiments are available at **[ChaosHub](http://hub.litmuschaos.io/)**.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/dan0AXO_soY?start=310" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/xLjTx8lqTuQ?start=1163" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Learn More
+- [What is a Chaos Workflow](../concepts/chaos-workflow)
```
Comment (S-ayanide) [inline website/docs/user-guides/construct-workflow.md]: Chaos Workflow and Chaos Engine are already covered as concepts the user needs to know thus you should put other concepts and user-guides relevant to this topic other than those.

**Review by S-ayanide — CHANGES_REQUESTED** [review]

**Comment by amityt** [comment]
Have made the requested changes @oumkale @S-ayanide . Thanks for the review!

**Comment by oumkale** [comment]
Thank you @amityt , For this PR!

**Review by S-ayanide — APPROVED** [review]

**Review by oumkale — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Pull request merged**